lib/HttpClient.php

```diff
@@ -256,11 +256,11 @@ public function _parse_and_convert_to_api_response($resp_content, $status_code,
 
         throw new \MagicAdmin\Exception\ApiException(
             '',
-            \array_key_exists('status', $resp_content) ? $resp_content->status : null,
+            \property_exists('status', $resp_content) ? $resp_content->status : null,
```

Comment:
The arguments passed to property_exists need to be reversed. Example: property_exists($resp_content, 'status')